### PR TITLE
Fixes #291:Replay and Continue button disappears on orientation change

### LINKED ISSUE #291: Replay and Continue button disappears on orientation change
**Current Behaviour**
replay and continue buttons in ScneraioOverActivity disappears on changing the device orientation.

ezgif com-optimize 6

**Expected Behaviour**
State of the buttons must me preserved on orientation change.

@chhavip @yatna @anubhakushwaha can I send PR for the same?

### PR DESCRIPTION
**Pull request by HimanshuS01**
Fixes #291 

ezgif com-optimize 6

@yatna @anubhakushwaha @chhavip @aanchal07 Please review this PR. Thanks :)

### CODE REVIEW

File: PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        android:background="@drawable/ic_launcher" />
+
+    <ImageView
+        android:id="@+id/health_text_ImageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_above="@+id/power_bar"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_marginRight="20dp"
```
Comment (aanchal07) [inline PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml]: Remove hard coding

File: PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        android:background="@drawable/brown_button_background"
+        android:text="@string/continue_text"
+        android:textAllCaps="false"
+        android:textSize="@dimen/button_font" />
+
+    <ImageView
+        android:id="@+id/karmaImageView"
+        android:layout_width="@dimen/karma_iv_width"
+        android:layout_height="@dimen/karma_iv_height"
+        android:layout_marginTop="5dp"
```
Comment (aanchal07) [inline PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml]: Remove hard coding

**Comment by HimanshuS01** [comment]
@anubhakushwaha @aanchal07 changes done. Please review this PR. Thanks :)

**Comment by HimanshuS01** [comment]
@chhavip conflicts has been resolved. Please review and merge this PR. Thanks :)

**Comment by chhavip** [comment]
Merging.

**Pull request merged**